pogs/pogs_test.go

```diff
@@ -1227,11 +1227,8 @@ func zfill(c air.Z, g *Z) error {
 	case air.Z_Which_anyList:
 		return c.SetAnyList(g.AnyList.ToPtr())
 	case air.Z_Which_anyCapability:
-		if g.AnyCapability == nil {
-			return c.SetAnyCapability(capnp.Ptr{})
-		}
 		cap := capnp.NewInterface(c.Segment(), c.Message().AddCap(g.AnyCapability))
-		return c.SetAnyCapability(cap.ToPtr())
+		return c.SetAnyCapability(cap.Client())
```

Comment:
I think this whole `case` could just be simplified to `c.SetAnyCapability(g.AnyCapability)`. This is the same logic that's moved into the setter.